lambdas/check-existing-identity/src/main/java/uk/gov/di/ipv/core/checkexistingidentity/CheckExistingIdentityHandler.java

```diff
@@ -184,36 +184,8 @@ public Map<String, Object> handleRequest(JourneyRequest event, Context context)
                     gpg45ProfileEvaluator.parseCredentials(
                             userIdentityService.getUserIssuedCredentials(userId));
 
-            boolean dataCorrelates = vcDataCorrelates(userId);
-            if (!dataCorrelates && completedF2F(f2fRequest, f2fVc)) {
-                var message =
-                        new StringMapMessage()
-                                .with(
-                                        LOG_MESSAGE_DESCRIPTION.getFieldName(),
-                                        "F2F return - failed to correlate VC data");
-                LOGGER.info(message);
-
-                auditService.sendAuditEvent(
-                        new AuditEvent(
-                                AuditEventTypes.IPV_F2F_CORRELATION_FAIL,
-                                configService.getSsmParameter(ConfigurationVariable.COMPONENT_ID),
-                                auditEventUser));
-
-                return JOURNEY_FAIL;
-            } else if (!dataCorrelates) {
-                LOGGER.info(
-                        new StringMapMessage()
-                                .with(
-                                        LOG_MESSAGE_DESCRIPTION.getFieldName(),
-                                        "VC data does not correlate so resetting identity."));
-
-                auditService.sendAuditEvent(
-                        new AuditEvent(
-                                AuditEventTypes.IPV_IDENTITY_REUSE_RESET,
-                                configService.getSsmParameter(ConfigurationVariable.COMPONENT_ID),
-                                auditEventUser));
-
-                return JOURNEY_RESET_IDENTITY;
+            if (!vcDataCorrelates(userId)) {
```

Comment:
@MikeCollingwood FYI you might get a minor merge conflict here (but easy to resolve :) )